1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -21,21 +21,35 @@ instead!
 3. Complete the unit test functions and verify that all is working as expected.
 -----------------------------------------------------------------------------*/
 const cartForParty = {
+  tea: 2.49,
+  cake: 8.99,
+  oranges: 2.59,
+  lemons: 0.99,
+  pears: 2.99,
   // TODO complete this object
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
+function calculateTotalPrice(foodForParty) {
+  let everything = 0;
+  for (const price of foodForParty) {
```

Comment:
When I run your code I'm the following error:
`TypeError: foodForParty is not iterable`

Reply:
That because we can can't do for loop on object but we can use `Object.keys()` instead.  

```suggestion
  Object.values(foodForParty).forEach((value) => {
    everything += value;
  });
```